index.html

```diff
@@ -8,7 +8,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap" rel="stylesheet">
     <script src="https://kit.fontawesome.com/1bc3a7e061.js" crossorigin="anonymous"></script>
-    <title>Mobile first Portfolio</title>
+    <title>My Portfolio | Belay Birhanu</title>
```

Comment:
- [ ] As mentioned by the previous reviewer, it would be better to use a different title according to accessibility. Instead of `My Portfolio | Belay Birhanu` you can say ` Belay Birhanu's Portfolio` 🙃 😉